tests/consensus.test.ts

```diff
@@ -0,0 +1,108 @@
+import { HandlerConstructorConfig } from "../types/handler";
+import { PrettyLogs } from "../types/logs";
+import nock from "nock";
+
+/**
+ * I had to separate this into it's own file as `nock` messed
+ * up the other tests.
+ *
+ * I didn't want to test the method like this, but I had to.
+ * The consensus function is naturally fragile when it comes
+ * to CI.
+ */
+const rpcUrls = ["http://127.0.0.1:8545", "http://127.0.0.1:8546", "http://127.0.0.1:8547"];
+
+export const testConfig: HandlerConstructorConfig = {
+  networkId: "100",
+  autoStorage: false,
+  cacheRefreshCycles: 3,
+  networkName: null,
+  rpcTimeout: 10000,
+  runtimeRpcs: rpcUrls,
+  networkRpcs: rpcUrls.map((url) => ({ url })),
+  proxySettings: {
+    retryCount: 3,
+    retryDelay: 10,
+    logTier: "verbose",
+    logger: new PrettyLogs(),
+    strictLogs: true,
+  },
+};
```

Comment:
this exact object is present in 3 files. Reduce duplication. Extract it to somewhere like tests/shared.ts and reuse it from here.